v2-components/Icons/ChevronUp/ChevronUp.tsx

```diff
@@ -0,0 +1,17 @@
+import { Icon, IconProps } from '@chakra-ui/react';
+
+interface ChevronUpProps extends IconProps {
+  width?: number | string;
+  height?: number | string;
+}
+
+export const ChevronUp = ({ width = '20px', height = '20px', ...props }: ChevronUpProps) => {
```

Comment:
There is already a ChevronUp icon from chakra ui and we are using that in the Paginator component. 
But as you said, worth taking a decision now on what we gonna use